lib/plugins/collaborators.js

```diff
@@ -114,7 +114,7 @@ module.exports = class Collaborators extends Diffable {
     } else {
       if (this.nop) {
         return Promise.resolve([
-          new NopCommand(this.constructor.name, this.repo, this.github.repos.removeCollaborator(
+          new NopCommand(this.constructor.name, this.repo, this.github.repos.removeCollaborator.endpoint(
```

Comment:
Out of curiosity: What is the method `endpoint` doing? I could not find it in the docs: https://github.com/octokit/plugin-rest-endpoint-methods.js/blob/master/docs/repos/removeCollaborator.md.

Reply:
It is a method in the request that returns the params and the url that would be called by that particular request method. There is an example [here](https://octokit.github.io/rest.js/v18#custom-requests)

Reply:
Just a dumb idea: How about making `NopCommand` making responsible for calling `.endpoint` in case `endpoint` argument is not null? It would require to provide the endpoint arguments as a separate parameter but at least you would ensure that NopCommand will be the only one calling these methods.

At least I can imagine that this bug might be introduced in the future again.

Reply:
Good idea, I'll make this change separately since it will affect the code broadly and we will need to test it more thoroughly